backend/.env

```
@@ -1,5 +1 @@
-DB_PASSWORD=D1ymf8wyQEGthFR1E9xhCq
-DB_USER=LTIdbUser
-DB_NAME=LTIdb
-DB_PORT=5432
-DATABASE_URL="postgresql://${DB_USER}:${DB_PASSWORD}@localhost:${DB_PORT}/${DB_NAME}"
\ No newline at end of file
+DATABASE_URL="postgresql://LTIdbUser:D1ymf8wyQEGthFR1E9xhCq@localhost:5432/LTIdb"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)